scripts/qa_crowbarsetup.sh

```diff
@@ -2908,6 +2908,18 @@ function onneutron_wait_for_neutron()
     fi
 }
 
+function get_nova_multi_controller_node()
```

Comment:
-1, this simply duplicates `get_novacontroller` (and does it wrong - it will fail in HA mode, see #477)